lib/shared/divider.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:thunder/shared/conditional_parent_widget.dart';
+
+class ThunderDivider extends StatelessWidget {
+  final bool sliver;

```

Comment:
I would add some comments here to indicate the parameters and what they mean! (especially `sliver` since that might not be obvious)

Reply:
Done!